test/backup.py

```diff
@@ -361,6 +365,21 @@ def _test_backup(self, tablet_type):
     # now bring up the other slave, letting it restore from backup.
     self._restore(tablet_replica2, tablet_type=tablet_type)
 
+    tablet_replica2.kill_vttablet()
+
+    # now delete the data dir
```

Comment:
Is it important for the test that there's partial data in the directory? I think `_restore()` is deleting everything anyway, so this deletion here probably has no effect.